common/lib/capa/capa/templates/textline.html

```diff
@@ -14,7 +14,7 @@
       <div style="display:none;" name="${hidden}" inputid="input_${id}" />
     % endif
 
-  <input type="text" name="input_${id}" id="input_${id}" aria-label="${label}" aria-describedby="answer_${id}" value="${value|h}"
+  <input type="text" name="input_${id}" id="input_${id}" aria-label="${response_data['label']}" aria-describedby="answer_${id} ${response_data['description_ids']}" value="${value|h}"
```

Comment:
@cahrens `aria-label="${response_data['label']}"` will be removed in https://github.com/edx/edx-platform/pull/13121. Didn't removed in this PR to avoid test failures.